src/components/__tests__/TimePicker.test.tsx

```diff
@@ -0,0 +1,150 @@
+import TimePicker from '../TimePicker'
+import React from 'react'
+import { render, screen } from '@testing-library/react'
+
+describe('TimePicker', () => {
+    const timePickerTestId = 'time-picker'
+
+    describe('no min/max entered', () => {
+        it('displays the correct time ranges with default time span', () => {
+            render(<TimePicker />)
+
+            const select = screen.getByTestId(timePickerTestId) as HTMLSelectElement
+
+            expect(select).toBeInTheDocument()
+
+            const options = extractOptions(select.options)
```

Comment:
I suspect you do not need this `extractOptions` helper. You should get the same (or almost the same thing) with:

```suggestion
            const options = screen.getAllByRole('option')
```

Reply:
When I originally had this in twist-web, I was able to just do `const options = [...select.options]`, but that wasn't available to me in reactist, something about es6 (I think).

Reply:
Oh ok, something to look into in how reactist is setup. But still, the alternative I shared should work regardless.